### DISCUSSION TITLE
Which ways are there to deploy a seleniumbase project

### DISCUSSION THREAD

**Discussion by mxaleding** [seleniumbase/SeleniumBase#2682] [General]
Hi,
I am kind of new to seleniumbase.
I wrote some short tests which are working fine on my local mashine. My Question is, which ways are there to deploy these test. I didn't find any tutorials for CI/CD or something similiar espacially for seleniumbase. Whould it be possible, to write an own simple web app using flask which triggers my test once a day?
Thank you for your help and Ideas!

**Comment by mdmintz** [comment]
There are various examples of CI/CD in this repo, such as [GitHub Actions](https://github.com/seleniumbase/SeleniumBase/blob/master/integrations/github/workflows/ReadMe.md).
